compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -3098,6 +3104,17 @@ TR::Register *OMR::Power::TreeEvaluator::vloadEvaluator(TR::Node *node, TR::Code
 
    TR::LoadStoreHandler::generateLoadNodeSequence(cg, dstReg, node, opcode, 16, true);
 
+   if (expandOpcode != TR::InstOpCode::bad &&
+       node->getDataType().getVectorElementType() == TR::Int8)
```

Comment:
such that, i have a further question: what happened running the same IL tree on pre-POWER10, i.e. the same `vload` node?

Reply:
Whoever generates mask load should've been doing the checking. But not problem now with vsrab.

Reply:
somewhere in the common code, do you mean that now you can generate this mask load on more platforms (previously for P10 only)?  do I expect to see this checking modification in this PR as well?

Reply:
The P10 check was missing but not needed now.

Reply:
great.